docs/.templates/jsonschemaforhumans/section_examples.md

```diff
@@ -1,4 +1,4 @@
-**Example{% if examples|length > 1 %}s{% endif %}:**{{ " " }}
```

Comment:
I thought this change was added in from the branched PR?

Reply:
No, I made this change to fix linting issues since this extra space isn't needed and adds an end of line space the linter doesn't like